packages/utils/src/date-utils/checkIsNextDay.ts

```diff
@@ -19,23 +21,23 @@ import { dayInMs } from './conversionUtils.js';
  * 22:00 - 02:00
  * 01:00 - 03:00
  */
-export function checkIsNextDay(previousStart: number, timeStart: number, previousDuration: number): boolean {
-  if (previousDuration === 0) {
+export function checkIsNextDay(
+  current: Pick<OntimeEvent, 'timeStart' | 'dayOffset'>,
+  previous?: Pick<OntimeEvent, 'timeStart' | 'duration' | 'dayOffset'> | null,
```

Comment:
question: does previous need to be both `undefined` and `null`?

Reply:
in the the timeline the lastEvent is marked as OntimeEvent or null, os it is to accommodate that...
or should I rater change how the timeline dose it?

Reply:
I think this would have to be your decision, ideally we want as few cases as possible